client/components/filters/index.js

```diff
@@ -46,7 +46,6 @@ class ReportFilters extends Component {
 			return (
 				<div className="woocommerce-filters__advanced-filters">
 					<AdvancedFilters
-						key={ JSON.stringify( query ) }
```

Comment:
Was this originally here to force some re-renders in this component tree?

Reply:
I was using it to [reset state](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key) and keep track of the previous parameters, but this was resetting all child components, including Search. This meant any saved queries were getting lost